app/components/topicPicker.js

```diff
@@ -67,66 +80,34 @@ export default React.createClass({
     // TODO: break this into its component pieces
     // const {topicId}: string = this.context.router.getCurrentParams();
 
+    console.log(this.props);
+
     return (
       <div className='topic-picker topic-picker--left'>
         <ul className='topic-picker__nav'>
-          <li data-topic='1'><b>Topic 1</b></li>
-          <li data-topic='2'><b>Topic 2</b></li>
-          <li data-topic='3'><b>Topic 3</b></li>
-          <li data-topic='4'><b>Topic 4</b></li>
+          { this.props.topicsTmp.map(topic => {
```

Comment:
it's found in getDefaultProps - it's just there for now to render something to show. it appears the data passed down into `this.props.topics` has a structure that moving forward will change, since it contains questions / answers which are not relevant for the topic picking view... at least this is how i understand it after talking to @nickbadams